### PR TITLE
Fix bug for extra spaces

### PR DESCRIPTION
**Pull request by cnbuff410**
Updated the slice allocation code but didn't update the assignment code.

### CODE REVIEW

**Comment by cnbuff410** [comment]
Will merge first, this has to be correct.

File: interpolation.go [hunk 1]
```diff
@@ -110,9 +110,9 @@ func InterP1(xList, vList []float64, queryList []float64) ([]float64, error) {
 
 	// Split
 	sortedX, sortedV := make([]float64, size), make([]float64, size)
-	for _, v := range p {
```
Comment (evanwht) [inline interpolation.go:113]: I thought append still worked here? The capacity is set to the expected size but the Len is still 0 so append should still insert values into the expected indices

Comment (cnbuff410) [reply interpolation.go:113]: The length is `size` by doing `make([]float64, size)`, so `append` would append value starting from `size` index.

Comment (cnbuff410) [reply interpolation.go:113]: `make([]float64, 0, size)` would do 0 length and `size` capacity

**Pull request merged**